frontend/src/app/tasks/actions/getTasksWithDate.ts

```diff
@@ -0,0 +1,32 @@
+import { makeGetTasks } from '@/infra/task/getTasks'
+import { serverFetch } from '@/infra/lib/serverFetch'
+import { authOptions } from '@/app/api/auth/[...nextauth]/route'
+import { getServerSession } from 'next-auth'
+
+import { Task } from '@/domain/Task'
+
+import { convertTimeToMinutes } from '../utils/time'
+
+export const getTasksWithDate = async (startTime: string, endTime: string, limit?: number) => {
```

Comment:
I'm a little confused about the naming, make it's better to call it `getTasksGroupedByDate` or something like that?